doc/admin-guide/files/records.config.en.rst

```diff
@@ -3087,6 +3087,10 @@ SSL Termination
 
    This configuration works with OpenSSL v1.1.1 and above.
 
+.. ts:cv:: CONFIG proxy.config.ssl.server.honor_cipher_order INT 1
+
+   By default (``1``) |TS| will use the server's cipher suites preferences instead of the client preferences.
```

Comment:
And if it's not `1`...? Then the client preference is used? I think it should be a bit more explicit if that's the case.

Reply:
Ack, updated!